tests/ApiFacadeTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Tests;
+
+use App\OxfordDictionary\ApiFacade;
+use App\OxfordDictionary\Clients\HttpClientInterface;
+use PHPUnit\Framework\TestCase;
+
+class ApiFacadeTest extends TestCase
+{
+
+    /**
+     * @test
+     */
+    public function throw_valid_exception()
```

Comment:
please use camelCase naming convention (throwValidExceptionTest)
and add Test to function name